### DISCUSSION TITLE
inline comment near else messes up

### DISCUSSION THREAD

**Discussion by epyu-lee** [PHP-CS-Fixer/PHP-CS-Fixer#6906] [Q&A] [answered]
Can anyone help me make this prettier?

Before:

```php
else
{ // text messages do not use a subject line.
    $send_this = true;
}
```

After:

```php
else // text messages do not use a subject line.
{$send_this = true;
}
```

Desired:

```php
else
{
    // text messages do not use a subject line.
    $send_this = true;
}
```

At a minimum, the curly brace should end up on a line by itself and everything properly indented.
 
 configuration:

```php
 return (new PhpCsFixer\Config())
    ->setRules([
        '@PSR2' => true,
        '@PhpCsFixer' => true,
        // New with v3, prevents writing "public" everywhere
        'visibility_required' => false,
        // TODO(lee): reenable this new rule later
        // during 8.0 upgrade it's triggering too many distracting changes.
        'single_line_comment_spacing' => false,
        // TODO: deprecated April 2023 use more detailed rules instead
        // https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/basic/braces.rst
        'braces' => ['position_after_control_structures' => 'next'],
        // 'concat_space' => ['spacing' => 'one'],
        'concat_space' => false,
        'yoda_style' => ['equal' => null, 'identical' => null],
        'blank_line_before_statement' => false,
        'ordered_class_elements' => ['order' => []],
        'no_unneeded_curly_braces' => false,
        // We don't want to turn off this elseif flag, as that will
        // change else if {} to else { if {}}.
        // 'elseif' => false,
        'phpdoc_align' => ['align' => 'left'],
    ])
```

**Comment by epyu-lee** [comment]
Still present in 3.17.0

**Comment by Wirone** [comment, marked as answer]
Hey!

Thanks for your report.

We identified that `braces` fixer is too complex and has too many responsibilities. With that, [we decided to deprecate it](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/4885).

That also means that we will not be extending, changing the behaviour or fixing anything around `braces` fixer. Please, use [replacement rules](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/basic/braces.rst#this-rule-is-deprecated-and-will-be-removed-on-next-major-version) instead.

If you see the issue that you reported to be present in one of the replacement rules, please report a new issue (you can also reference this one if you think past discussions have meaningful context).

Big thanks!

**Comment by epyu-lee** [comment]
I'm seeing precisely the same behavior the new way.  That's why I switched to the new way--to fix this.  Sorry I didn't update this question.

Do you have a fiddling environment I can use to demonstrate the problem independent of an other MSVS code features that might be clouding the issue?
